package.json

```diff
@@ -11,7 +11,7 @@
     "lib/"
   ],
   "scripts": {
-    "prepare": "rimraf lib && npm run build",
+    "prebuild": "rimraf lib",
```

Comment:
@balanza not sure about it?

Reply:
uselss imho. `tsc` is handling this already